app/helpers/pages_helper.rb

```diff
@@ -11,4 +19,10 @@ def page_breadcrumb_links(page)
       "<div class='link'>#{link}</div>"
     end.join
   end
+
+  def term_name
+    today = Date.today
+    term = 2 < today.month || today.month < 9 ? 's' : 'f'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.